src/pages/ContentScripts/features/repo-fork-tooltip/view.tsx

```diff
@@ -37,7 +37,7 @@ const View = ({ forks, meta }: Props): JSX.Element | null => {
       <ForkChart
         theme={githubTheme as 'light' | 'dark'}
         width={270}
-        height={130}
+        height={150}
```

Comment:
Why change the height of the fork tooltip to 150 while keep height of others unchanged?

Reply:
I accidentally changed it and forgot to change it back. Afterwards, I will make the necessary changes back.